packages/scripts/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@molgenis/scripts",
+  "version": "0.1.0",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "build-watch": "vue-cli-service build --watch",
+    "e2e": "vue-cli-service test:e2e",
```

Comment:
why are these not the default vue cli targets ? suggest to use is the defaults unless this is not possible

Reply:
The default Lerna/Jenkinsfile in the root expects this calling convention.